_fr/getting-started/index.md

```diff
@@ -0,0 +1,194 @@
+> $ cs install scala3
+> ```
+
+### ...ou manuellement
+
+Vous avez seulement besoin de deux outils pour compiler, lancer, tester et packager un projet Scala: Java 8 ou 11, et sbt.
+Pour les installer manuellement :
+
+1. Si vous n'avez pas Java 8 ou 11 installé, téléchargez
+   Java depuis [Oracle Java 8](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html), [Oracle Java 11](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html),
+   ou [AdoptOpenJDK 8/11](https://adoptopenjdk.net/). Référez-vous à la page [JDK Compatibility](/overviews/jdk-compatibility/overview.html) pour les détails de compatibilité entre Java et Scala.
+1. Installez [sbt](https://www.scala-sbt.org/download.html)
+
+## Créer un projet "Hello World" avec sbt
+
+Une fois que vous avez installé sbt, vous pouvez créer un projet Scala, comme expliqué dans la section suivante.
+
+Pour créer un projet, vous pouvez soit utiliser la ligne de commande, soit un IDE.
+Si vous êtes habitué à la ligne de commande, nous recommandons cette approche.
+
+### Utiliser la ligne de commande
+
+sbt est un outil de construction de package pour Scala, sbt compile, lance et teste votre code Scala.
+(Il peut aussi publier les librairies et faire beaucoup d'autres tâches.)
+
+Pour créer un nouveau projet Scala avec sbt :
+
+1. `cd` dans un répertoire vide.
+1. Lancez la commande `sbt new scala/scala3.g8` pour créer un projet Scala 3, ou `sbt new scala/hello-world.g8` pour créer un projet Scala 2.
+   Cela va télécharger un projet template depuis Github.
+   Cela va aussi créer un dossier `target`, que vous pouvez ignorer.
+1. Quand cela vous est demandé, nommez votre application `hello-world`. Cela va créer un projet appelé "hello-world".
+1. Voyons ce que nous vennons de générer :
+
+```
+- hello-world
+    - project (sbt utilise ce dossier pour ses propres fichiers)
+        - build.properties
+    - build.sbt (fichier de définition de la construction du package pour sbt)
+    - src
+        - main
+            - scala (tout votre code Scala doit être placé ici)
+                - Main.scala (Programme point d'entrée) <-- c'est tout ce dont nous avons besoin pour le moment
+```
+
+Vous pouvez trouver plus de documentation à propos de sbt dans le [Scala Book](/scala3/book/tools-sbt.html) ([Lien](/overviews/scala-book/scala-build-tool-sbt.html) vers la version Scala 2) et sur la [documentation](https://www.scala-sbt.org/1.x/docs/index.html) officielle de sbt.
+
+### Avec un IDE
+
+Vous pouvez ignorer le reste de cette page et aller directement sur [Building a Scala Project with IntelliJ and sbt](/getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.html).
+
+
+## Ouvrir le projet hello-world
+
+Utilisons un IDE pour ouvrir le projet. Les plus populaires sont IntelliJ et VSCode.
+Il proposent tout deux des fonctionnalités riches, mais vous pouvez utiliser d'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html)
```

Comment:
I find the second part of the sentence "..., mais vous pouvez utiliser d'autres éditeurs." a bit weird, as it reads like a restriction or negation.

```suggestion
Il proposent tout deux des fonctionnalités riches. D'[autres éditeurs.](https://scalameta.org/metals/docs/editors/overview.html) sont également disponibles.
```